src/utils/frame.ts

```diff
@@ -0,0 +1,128 @@
+import calculateCrc16 from './calculateCrc16.js';
+import getBytesFromHex from './getBytesFromHex.js';
+import getBytesFromBase64 from './getBytesFromBase64.js';
+import BinaryBuffer from './BinaryBuffer.js';
+
+
+const STUFFING_BYTE = 0x7d;
+
+const byteStuff = ( byte: number ): number => {
+    const byteStuffMap: Record<number, number> = {0x13: 0x33, 0x11: 0x31, 0x7d: 0x5d, 0x7e: 0x5e};
+
+    return byteStuffMap[byte] || byte;
+};
+
+const byteUnstuff = ( byte: number ): number => {
+    const byteUnstuffMap: Record<number, number> = {0x33: 0x13, 0x31: 0x11, 0x5d: 0x7d, 0x5e: 0x7e};
```

Comment:
the same

and the function body is very small now

Reply:
fixed